include/dbclean.php

```diff
@@ -18,16 +23,20 @@ function dbclean_run(&$argv, &$argc) {
 		unset($db_host, $db_user, $db_pass, $db_data);
 	}
 
-	load_config('config');
-	load_config('system');
```

Comment:
Yeah, the config class was created some time ago and we want to replace the old calls with the class calls. Is this some task that you would like to do?